lib/bolt/shell/powershell.rb

```diff
@@ -274,7 +274,12 @@ def run_task(task, arguments, _options = {}, position = [])
                               []
                             end
 
-          output = execute([Snippets.shell_init, *env_assignments, command].join("\n"))
+          output = execute([
+            Snippets.shell_init,
+            Snippets.append_ps_module_path(task_dir),
```

Comment:
See comment for `#append_ps_module_path`. If making that change, this would be updated to take `dir` and `extra_files` as parameters.